R/create_single_dose_dataset.R

```diff
@@ -0,0 +1,272 @@
+#' Pre-Defined Dose Frequencies
+#'
+#' These pre-defined dose frequencies are sourced from
+#' [CDISC](https://evs.nci.nih.gov/ftp1/CDISC/SDTM/SDTM%20Terminology.pdf). The
+#' number of rows to generate using `create_single_dose_dataset()` arguments
+#' `start_date` and `end_date` is derived from `DFFactor` and `DFUnit` with
+#' appropriate functions from `lubridate`.
+#'
+#' @details
+#' To see the entire table in the console, run `print(dose_freq_lookup)`.
+#'
+#' @seealso [create_single_dose_dataset()]
+#'
+#' @export
+#'
+#' @keywords dose_freq
+#'
+#' @rdname dose_freq_lookup
+#'
+dose_freq_lookup <- tibble::tribble(
+  ~NCICode, ~CDISCSubmissionValue,
```

Comment:
@thomas-neitmann , @bms63 , what is our convention for dataset variable names? Should they be uppercase or does this apply to ADaM datasets only?